icicle/appUtils/ntt/ntt_impl.cuh

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#ifndef _LARGE_NTT_H
+#define _LARGE_NTT_H
+
+#include <stdint.h>
+#include "appUtils/ntt/ntt.cuh" // for enum Ordering
+
+namespace ntt {
+
+  template <typename S>
+  cudaError_t generate_external_twiddles_generic(
```

Comment:
Why are these declarations separated from ntt.cuh?

Reply:
because ntt.cuh is an external header but those declarations are not supposed to be exposed.
If/when we install the library this header is not supposed to be installed.